website/server/controllers/api-v3/challenges.js

```diff
@@ -21,18 +21,153 @@ import csvStringify from '../../libs/csvStringify';
 let api = {};
 
 /**
+ * @apiDefine ChallengeLeader Challenge Leader
+ * The leader of the challenge can use this route.
+ */
+
+/**
  * @apiDefine ChallengeNotFound
  * @apiError (404) {NotFound} ChallengeNotFound The specified challenge could not be found.
  */
 
 /**
+ * @apiDefine SuccessfulChallengeRequest
+ * @apiSuccess {UUID} challenge.group._id The group id.
+ * @apiSuccess {String} challenge.group.type Group type: `Guild` or `Party`.
+ * @apiSuccess {String} challenge.group.privacy Group privacy: `Public` or `Private`.
+ * @apiSuccess {String} challenge.name Full name of challenge.
+ * @apiSuccess {String} challenge.shortName Short name of challenge.
+ * @apiSuccess {Object} challenge.leader User details of challenge leader.
+ * @apiSuccess {UUID} challenge.leader._id User id of challenge leader.
+ * @apiSuccess {Object} challenge.leader.profile Profile information of leader.
+ * @apiSuccess {Object} challenge.leader.profile.name Username of leader.
```

Comment:
"Username" should be "Display name" or "Profile name", probably the former since we already have profile mentioned in the attribute itself; both are commonly used for `profile.name`
"Username" is more often used to refer to the login name.